src/frontends/paddle/tests/test_models/gen_scripts/generate_embedding.py

```diff
@@ -12,20 +12,20 @@
 
 from save_model import saveModel
 
-def ngraph_embedding(ids, vocab_embeddings, vocab_size, embedding_dim, padding_idx, sparse):
+def ov_embedding(ids, vocab_embeddings, vocab_size, embedding_dim, padding_idx, sparse):
     """
     decomposing embedding with OpenVINO ops.
     """
-    import ngraph as ng
-    from ngraph import opset8 as opset
-    from openvino.inference_engine import IECore
+    import openvino as ov
+    import openvino.runtime.opset8 as ov_opset8
```

Comment:
```suggestion
     from openvino.runtime import opset8
```
and later use `opset8` instead of `ov_opset8`

Reply:
Thanks, fixed!